include/erlcloud_ecs.hrl

```diff
@@ -188,6 +193,7 @@
     memory :: undefined | pos_integer(),
     memory_reservation :: undefined | pos_integer(),
     mount_points :: undefined | [#ecs_mount_point{}],
+    placement_strategy :: undefined | [#ecs_placement_strategy{}],
```

Comment:
where is the code to actually set this to something other than default? I don't see it in this PR.

Reply:
It's a [run_task option](https://github.com/erlcloud/erlcloud/pull/547/files#diff-67e666f62a38974173c59f4bcd66bb1dR2175)